### DISCUSSION TITLE
TODO: Use refraction-networking/utls to Imitate TLS Fingerprints

### DISCUSSION THREAD

**Discussion by DuckSoft** [Qv2ray/gun#21] [Ideas]
It seemed that Golang's default TLS fingerprint has been explicitly blocked in some countries.
To accommodate these users, let's try introduce uTLS (or similar technology) to bypass the limit.

cc @rprx @xiaokangwang

**Comment by xiaokangwang** [comment]
So... the final result is that browser forwarding will be used to solve this issue. I believe Chrome already support streaming upload body.
